omnibus/files/private-chef-cookbooks/private-chef/recipes/redis_lb.rb

```diff
@@ -130,14 +134,19 @@
 
     redis.pipelined do
       # Now we're clear to repopulate from configuration.
-      unless banned_ips.nil?
-        banned_ips.each do |ip|
-          redis.sadd   'banned_ips', ip
-        end
-      end
-      unless maint_mode_ips.nil?
-        maint_mode_ips.each do |ip|
-          redis.sadd   'maint_data', ip
+#       unless banned_ips.nil?
+#         banned_ips.each do |ip|
+#           redis.sadd   'banned_ips', ip
+#         end
+#       end
+#       unless maint_mode_ips.nil?
+#         maint_mode_ips.each do |ip|
+#           redis.sadd   'maint_data', ip
+#         end
+#       end
+      unless xmaint_whitelisted_ip.nil?
+        xmaint_whitelisted_ip.each do |ip|
+          redis.sadd   'maint_whitelisted_ip', ip
```

Comment:
This is perfectly fine, and there is absolutely nothing wrong with it.  You don't need to change anything.  This is just "for your information" if you don't know and are interested.  Since Ruby 2.3 (I'm not sure what version of Ruby we are on - we would need to check) there is a 'safe navigator' operator `&.`.  If the object is `nil`, it just returns `nil` and doesn't do anything.  But if the object isn't `nil`, it executes the method.  It saves you from having to check if the object is `nil` before doing something.  Here is a demonstration:
```
irb(main):036:0>  [-1, -2, -3]&.each do |i| puts i.abs end
1
2
3
irb(main):037:0> nil&.each do |i| puts i.abs end
=> nil
```
Also, if it's a short one-liner like yours, you can also remove the `do...end` and use braces:
```
irb(main):038:0> [-1, -2, -3]&.each{|i| puts i.abs}
1
2
3
=> [-1, -2, -3]
```